services/types.go

```diff
@@ -1,7 +1,16 @@
 package services
 
+type Role string
+
+const (
+	ClientRole        Role = "client"
+	MirroringPeerRole Role = "mirroring-peer"
+)
+
 type OnboardingTicket struct {
 	ID                string `json:"id"`
 	ExpirationDate    int64  `json:"expirationDate,string"`
 	StorageQuotaInGiB uint   `json:"storageQuotaInGiB,omitempty"`
+	// Role can be either client or mirroring-peer
+	Role Role `json:"role,string"`
```

Comment:
The rule is not for the token, it is for the subject. 
```suggestion
	SubjectRole Role `json:"subjectRole,string"`
```